internal/types/responses.go

```diff
@@ -35,3 +35,8 @@ type ListResponse[T any] struct {
 	Rows       []T                `json:"rows"`
 	Pagination PaginationResponse `json:"pagination"`
 }
+
+// ResponseWithTaskNames defines the structure for a successful response with task names
+type ResponseWithTaskNames struct {
```

Comment:
This should be in `pkg` since it is an API response. 
Probably with the other task RPC types.

Reply:
anything in `internal` can't be imported by other projects so this type wouldn't be accessible if Talis was used in a test by an external repo.

Reply:
great catch. thanks!